### PR TITLE
[SETUP] Config Vuetify

### PR DESCRIPTION
**Pull request by rartner**

### CODE REVIEW

**Comment by rartner** [comment]
[Configurar Vuetify](https://trello.com/c/GBuwWsWt/28-configurar-vuetify)

File: app/javascript/packs/dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import App from './components/app.vue'
+import Vuetify from "vuetify";
```
Comment (cuzik) [inline app/javascript/packs/dashboard/index.js]: single quotes

File: app/javascript/packs/dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import colors from "vuetify/es5/util/colors";
```
Comment (cuzik) [inline app/javascript/packs/dashboard/index.js]: single quotes

File: app/javascript/packs/dashboard/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Vue.use(Vuetify);
+
+document.addEventListener('DOMContentLoaded', () => {
+  const el = document.querySelector("#dashboard");
+  const app = new Vue({ 
+    el,
+    render: h => h(App),
+  });
+})
```
Comment (cuzik) [inline app/javascript/packs/dashboard/index.js]: add empty line

**Pull request merged**